core/server/themes/validate.js

```diff
@@ -1,21 +1,31 @@
 var Promise = require('bluebird'),
+    gscan = require('gscan'),
```

Comment:
See https://github.com/TryGhost/Ghost/pull/8121/files#diff-8d96493a641ecbe40ffd7f378b4a175fL6
Would be great to not require gscan on bootstrap. Eats too much un-needed resources.